### PR TITLE
Update deploy

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-provision/pull/13
https://github.com/open-orchestra/open-orchestra/pull/546

### CODE REVIEW

**Comment by nicolasThal** [comment]
Could you also add a hook to restart apache and varnish after deploy:finished ?

**Comment by itkg-ngilain** [comment]
@nicolasThal : something like this ?

File: deployment/deploy.rb [hunk 1]
```diff
@@ -9,3 +9,5 @@
 set :linked_dirs, %w{app/logs web/uploads vendor bower_components node_modules}
 
 after 'deploy:finishing', 'deploy:cleanup'
+after 'deploy:finishing', 'web:apache'
```
Comment (nicolasThal) [inline deployment/deploy.rb]: I would rather put those line in the task definition,

**Comment by itkg-ngilain** [comment]
@nicolasThal : update done

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**